templates/base.html

```diff
@@ -152,9 +152,6 @@
         {{ js('common') }}
         <script async defer src="{{ cache_buster(url('addons.buttons.js')) }}"></script>
       {% endblock %}
-      {% block tabzilla_js %}
-        <script src="https://mozorg.cdn.mozilla.net/{{ LANG }}/tabzilla/tabzilla.js"></script>
```

Comment:
So, no more `tabzilla.js` at all? Excellent. That's one less copy of jQuery to worry about.